src/Feedback-Service/Dtos/CreateFeedbackDto.cs

```diff
@@ -1,8 +1,25 @@
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.InteropServices;
+
 namespace Feedback_Service.Dtos;
 
-public class CreateFeedbackDTO
+public class CreateFeedbackDto
 {
-    public Guid PatientId { get; set; }
+    public int PatientId { get; set; }
+    public Guid AuthorId { get; set; }
     public Guid StressMeassurementId { get; set; }
```

Comment:
I believe you made a small typo here: `StressMeassurementId` should be `StressMeassurementId`